core/docker/docker-compose.yml

```diff
@@ -37,13 +37,15 @@ services:
       - hapi.fhir.allow_external_references=true
       - hapi.fhir.max_page_size=50
       - hapi.fhir.default_page_size=20
+      - "JAVA_TOOL_OPTIONS=-Xmx2g"
+      - "CATALINA_OPTS=-Xmx2g"
     volumes:
       - type: volume
         source: instant
         target: /instant
     depends_on:
       - mysql
-
+ 
```

Comment:
Please remove trailing whitespaces - it's bad practice to commit code clutter like this when a linter would have removed them

```suggestion

```